src/jvm.jl

```diff
@@ -246,6 +247,7 @@ function destroy()
     if (!isdefined(JavaCall, :penv) || penv == C_NULL)
         throw(JavaCallError("Called destroy without initialising Java VM"))
     end
+	assertroottask()
```

Comment:
indentation

Reply:
Fixed in 0614efa . There was another [tab](https://github.com/JuliaInterop/JavaCall.jl/pull/105/commits/0614efa89d1bc67e46bd80d96f9070af6a42f87c#diff-72e942232acb05e3dc6f606c0a05a86fL71) in the file as well.

Reply:
I fixed additional indentation issues in JavaCall.jl due to tab characters: de0bfe5